blueprints/ember-electron/index.js

```diff
@@ -45,12 +46,54 @@ module.exports = class EmberElectronBlueprint extends Blueprint {
   }
 
   async afterInstall() {
+    await this.updateEnvironmentConfig();
     await this.updateTravisYml();
     await this.updateEslintIgnore();
     await this.updateEslintRc();
     await this.createElectronProject();
   }
 
+  async updateEnvironmentConfig() {
+    this.ui.writeLine(chalk.green('Updating config/environment.js'));
+
+    let contents = (await readFile('config/environment.js')).toString();
+
+    const rootURLRegex = /(\srootURL:)/;
+    if (rootURLRegex.test(contents)) {
+      contents = contents.replace(rootURLRegex, `$1 process.env.EMBER_CLI_ELECTRON ? '' :`);
+    } else {
+      this.ui.writeLine(chalk.yellow([
+        `\nUnable to update rootURL setting to`,
+        `\`process.env.EMBER_CLI_ELECTRON ? '' : <previous value>\`,`,
+        `which is needed for your Ember app to load assets under Electron.`,
+        `See ${routingAndAssetLoadingUrl} for more information.`
+      ].join(' ')));
+    }
+
+    const locationTypeRegex = /(\slocationType:)/;
```

Comment:
This is a bit pedantic, but whitespace is allowed between the key name and the `:` so it might be worth adding `\s?` just before to cover that case also.

```js
[" foo:123", " foo : 123" ].map(s => /(\sfoo:)/.test(s)) // --> [ true, false ]
[" foo:123", " foo : 123" ].map(s => /(\sfoo\s?:)/.test(s)) // --> [ true, true ]
```

Reply:
Yeah, I suppose. I was mainly trying to target the output of the default app blueprints. The leading `\s` was to make sure that the `locationType` wasn't a substring, like if the user had another setting that was `colocationType` or something...probably even less likely than the user putting spaces before the `:` 😄 I don't want to go crazy trying to handle all kinds of cases of modifications to the app-blueprint-generated, but I think that extra `\s*` is super reasonable -- I'll add it.

Reply:
Got to draw the line somewhere, but for me that point is more like supporting `{ [eval("'foo'")]: ...}` 😱